fe/README.md

```diff
@@ -1,16 +1,30 @@
-# smart_turner
+# SmartTurner - Running the Flutter App
 
-A new Flutter project.
 
-## Getting Started
+## Install backend dependencies
+```
+cd be
+npm install --immutable
+```
 
-This project is a starting point for a Flutter application.
+## Run the server
+Inside `/be`, create a `.env` file with the contents `NODE_PORT=3000`. Now we can run the server at port 3000:
+```
+npm run dev
+```
+
+## Run the Flutter app on the Web (Chrome)
+Open a new terminal window and navigate to the `/fe` directory.
+
+Inside `/fe`, create a directory called `/assets`, if it does not already exist. 
+
+Add `happy_birthday.mxl` to this `/assets` directory, if it does not already exist.
+> You can also upload any other `.mxl` file of your choosing, just make sure to change the `filename` in `main.dart` to be the name of your file:
+```
+const filename = "happy_birthday.mxl";
+```
+
+Select the Web as your device and run the Flutter app!
```

Comment:
You can change this to something akin to:

To run this Flutter app, either connect to an iPad or Android tablet device, or start up a tablet emulator on your development machine, then run `flutter run`.